.storybook/main.js

```diff
@@ -1,14 +1,4 @@
 module.exports = {
   stories: ['../styleguide/**/*.stories.@(js|mdx)'],
-  addons: [
-    {
-      name: '@storybook/addon-essentials',
```

Comment:
I have seen others use the viewport tool this provides, might be worth leaving it for now?

Reply:
Ah good shout. I'm mostly trying to avoid the churn of having the whole essentials module. Looks like we can pull in https://storybook.js.org/addons/@storybook/addon-viewport/